samples/kctf-conf/base/Makefile

```diff
@@ -19,7 +19,7 @@
 # See the License for the specific language governing permissions and
```

Comment:
This is the same exact file as `base/Makefile`, right? Why it's not a symlink like in `kctf/base/challenge-skeleton/Makefile`? Should I also duplicate changes in two Makefiles in my PR?

Reply:
oh, yea.. /samples/ is the equivalent of what an "upgrade" in infrastructure would be. this means, every CL needs to update /samples/, but we always forget :(. we can't make it a symlink because in "real life" users wouldn't do this.

the way I usually do the "upgrade" is that I delete the `kctf-conf/base` directory and then run setup-chal-dir, which recreates it from "head".